bin/elixir

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 set -e
 
-if [ $# -eq 0 ] || [ "$1" = "--help" ] || [ "$1" = "-h" ]; then
+if [ $# -eq 0 -o "$1" = "--help" -o "$1" = "-h" ]; then
```

Comment:
Please revert these changes. `||` is the style we use in these files and we shouldn't change it. Please use `&&` below.

Reply:
To be clear, we shouldn't change it in this PR. We can do it in a separate one.

Reply:
This requires to spawn multiple `test` instances on each run of the application, that is why I introduced this change.